src/server.h

```diff
@@ -179,7 +179,7 @@ typedef struct redisObject robj;
 
 #define REPLY_BUFFER_DEFAULT_PEAK_RESET_TIME 5000 /* 5 seconds */
 
-/* When configuring the server eventloop, we setup it so that the total number
+/* When configuring the server eventloop, we set up it so that the total number
```

Comment:
I think setup better then set up

Reply:
setup: noun
set up: verb

Reply:
> setup: noun set up: verb
All right.

Reply:
it should probably read "we set it up so that"